doc/source/ray-observability/key-concepts.rst

```diff
@@ -75,11 +75,12 @@ Profiling
 ---------
 Profiling is way of analyzing the performance of an application by sampling the resource usage of it. Ray supports various profiling tools:
 
-- CPU profiling for Worker processes, including integration with :ref:`py-spy <dashboard-profiling>` and :ref:`cProfile <dashboard-cprofile>`
-- Memory profiling for Worker processes with :ref:`memray <memray-profiling>`
+- CPU profiling for Driver and Worker processes, including integration with :ref:`py-spy <dashboard-profiling>` and :ref:`cProfile <dashboard-cprofile>`
```

Comment:
Should we have a link to the new profiling.md page here?

Reply:
Done